README.md

```diff
@@ -1,5 +1,5 @@
-To assemble a jar file for your project, run the "distZip" gradle task, either through IntelliJ or by doing
-`gradle distZip` on a terminal. Gradle will automatically download all dependencies needed to compile your zip file,
+To assemble a jar file for your project, run the "shadowJar" gradle task, either through IntelliJ or by doing
```

Comment:
Why the shadow jar? The distZip creates a zip that can then be extracted and the jar inside can be run with all the proper dependencies